react/features/base/redux/functions.js

```diff
@@ -127,8 +127,6 @@ export function persistState(state: Object) {
         PERSISTED_STATE_NAME,
         JSON.stringify(filteredState)
     );
-
-    logger.info('Redux state persisted');
```

Comment:
Why this log is removed?

Reply:
It logs too much (approx every 2 seconds) and doesn't make too much sense

Reply:
shouldn't we persist only if this state has changed ?

Reply:
Should we do a diff on every state change?

Reply:
no, but we filter the state that is to be *eventually* persisted, then it's not so much to diff i f we diff only this part.

Reply:
I mean writing the same thing every 2 seconds doesn't make sense. Especially on mobile

Reply:
But is it related to this issue somehow?

Reply:
If not - let's move it to another PR or at least another commit.

Reply:
no, it should be done is separate PR

Reply:
yeah I guess it's fine that it will log that every 2 seconds so that we won't forget it's actually doing that

Reply:
Ok, so should I put it back?

Reply:
yes please